### PR TITLE
patch seems to work

### PR DESCRIPTION
**Pull request by r4vi**

### CODE REVIEW

**Comment by r4vi** [comment]
Is this what you were trying to do?

**Comment by xoen** [comment]
I'm 99% at some point I tried that. Let me try again and if it does I'll just use `patch` as it's less hacky than changing a module variable from the outside (semantics really). Thanks @r4vi 🌟

**Comment by xoen** [comment]
FYI: Closing this. See: https://github.com/ministryofjustice/analytics-platform-idler/pull/68/commits/8f270b47f6f22b6f4c9bd10f6fe7e198a90575fa
